src/main/java/teammates/ui/controller/FeedbackSubmissionEditPageData.java

```diff
@@ -20,6 +22,7 @@
     public boolean isHeaderHidden;
     public StudentAttributes studentToViewPageAs;
     public InstructorAttributes previewInstructor;    
+    private String registerMessage;
```

Comment:
previewInstructor is used in InstructorFeedbackPreviewAsInstructorAction.java and instructorFeedbackSubmissionEdit.jsp